Source/Core/DolphinQt/TAS/IRWidget.h

```diff
@@ -34,5 +34,7 @@ public slots:
 };
 
 // Should be part of class but fails to compile on mac os
+static const u16 ir_min_x = 0;
+static const u16 ir_min_y = 0;
 static const u16 ir_max_x = 1023;
 static const u16 ir_max_y = 767;
```

Comment:
Note that these can be in the class (at least if they're changed to `constexpr`). This is a change I make in the balance board PR (#8268).